lib/karaokium_web/live/admin/karaoke_live/show.html.heex

```diff
@@ -15,7 +15,7 @@
   <h1>
     <%= @karaoke.name %>
   </h1>
-  <%= raw(qrcode(Routes.karaoke_show_url(@socket, :show, @karaoke))) %>
+  <%= raw(qrcode(Routes.page_url(@socket, :index, %{"karaoke" => %{"code" => @karaoke.code}}))) %>
```

Comment:
I did this because the QRCode will change everytime the reset pin changes. That way if someone takes a picture and the PIN changes, the QR code is useless. There are some concerns still but this app is not so critical for us to worry about.